template.json

```diff
@@ -40,7 +41,8 @@
       "lint:eslint": "eslint ./src --ext .js,.jsx,.ts,.tsx --cache",
       "fix": "npm-run-all fix:*",
       "fix:eslint": "npm run lint:eslint -- --fix",
-      "fix:prettier": "prettier \"src/**/*.js,jsx,ts,tsx,scss}\" --write"
+      "fix:prettier": "prettier \"src/**/*.js,jsx,ts,tsx,scss}\" --write",
+      "deploy": "cross-env rsync -avPx build/* ${npm_package_name}@<host>:~/public/ --delete-after"
```

Comment:
I usually call this one `upload` and have `deploy` that calls `yarn build && yarn upload && yarn open`

Reply:
I've added `deploy` and `upload-build`